src/lib/AutomateSDK.ts

```diff
@@ -226,6 +237,15 @@ export class AutomateSDK {
     overrides: Overrides = {},
     creatorAddress?: string
   ): Promise<CreateTaskPopulatedTransaction> {
+    //TODO: Temporary zksync fix
+    const { isDeployed } = await this.getDedicatedMsgSender();
```

Comment:
The `prepare` method is not supposed to send any transactions as it would break for users using the sdk just to prepare tasks without importing the correct `signer` 

Maybe what we could do is:
- `prepareBatchExecTask `: throw an error if the contract is not deployed?
- `createBatchExecTask`: force the deployment before creating the task